### PR TITLE
DOCS PHP 7.4 support

### PR DESCRIPTION
**Pull request by brynwhyman**

### CODE REVIEW

File: docs/en/04_Changelogs/4.6.0.md [hunk 1]
```diff
@@ -8,6 +8,7 @@
  * [Solr no longer indexes draft/restricted content](#solr-updates)
  * [Simplify customisation of ModelAdmin](#modeladmin-customisation)
  * [Login forms module ships with installer](#loginforms)
+ * [PHP 7.4 Compatibility](#php7-4)
```
Comment (robbieaverill) [inline docs/en/04_Changelogs/4.6.0.md:11]:
```suggestion
 * [PHP 7.4 compatibility](#php7-4)
```

File: docs/en/04_Changelogs/4.6.0.md [hunk 2]
```diff
@@ -334,6 +335,10 @@ login form.
 
 View the [Silverstripe Login Forms readme](https://github.com/silverstripe/silverstripe-login-forms) on Github for more details.
 
+## PHP 7.4 Compatibility {#php7-4}
```
Comment (robbieaverill) [inline docs/en/04_Changelogs/4.6.0.md:338]:
```suggestion
## PHP 7.4 compatibility {#php7-4}
```

**Comment by brynwhyman** [comment]
This is based off the work that went into adding PHP 7.4 to the cwp-recipe-kitchen-sink test matrix, captured in `2.6`/ `4.6`: https://github.com/silverstripe/cwp-recipe-kitchen-sink/commit/7eb5ea37bf96c71b18fdf084d16e9a6c14713f77 ([parent issue](https://github.com/silverstripe/cwp-recipe-kitchen-sink/issues/57)).

You're right though, based on this issue: https://github.com/silverstripe/silverstripe-framework/issues/9116 PHP 7.4 support should be assumed for `4.5` as well. PHP 7.4 builds have also passed in the `4.5` branch: https://travis-ci.org/github/silverstripe/silverstripe-installer/builds/655204320

Are we safe to update this PR to record 7.4 support from 4.5?

**Comment by robbieaverill** [comment]
Maybe you could denote CWP as different to core in these changes, that might help?

**Comment by brynwhyman** [comment]
Alright, updated the PR. How does that look?

**Pull request merged**